src/Controller/UserController.php

```diff
@@ -31,9 +31,20 @@ function connect(Application $app) {
 
     function indexAction(Application $app) {
         $users = $app['service.user']->all();
+        $usersData = [];
+
+        foreach ($users as $user) {
+            $calendars = count($app['sabredav.backend.caldav']->getCalendarsForUser($user->getPrincipalUri()));
+            $addressbooks = count($app['sabredav.backend.carddav']->getAddressBooksForUser($user->getPrincipalUri()));
+            $usersData[] = [
+                'userInfo'     => $user,
+                'calendars'    => $calendars,
```

Comment:
should be called `calendarCount` to not confuse people into thinking its an array.